### PR TITLE
2 packages from abenori/satysfi-pagestyle at 1.0.0

### PR DESCRIPTION
**Pull request by abenori**
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [ ] Add to snapshot `snapshot-develop` :: satysfi-pagestyle-doc.1.0.0 satysfi-pagestyle.1.0.0
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- [ ] Add to snapshot `snapshot-stable-0-0-5` :: satysfi-pagestyle-doc.1.0.0 satysfi-pagestyle.1.0.0
- [ ] Add to snapshot `snapshot-stable-0-0-6` :: satysfi-pagestyle-doc.1.0.0 satysfi-pagestyle.1.0.0

### CODE REVIEW

File: packages/satysfi-pagestyle-doc/satysfi-pagestyle-doc.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Brilliant description comes here.
+"""
+maintainer: "Noriyuki Abe <abenori@users.osdn.me>"
+authors: "Noriyuki Abe <abenori@users.osdn.me>"
+license: "BSD-2-Clause-FreeBSD"
+homepage: "https://github.com/abenori/satysfi-pagestyle"
+dev-repo: "git+https://github.com/abenori/satysfi-pagestyle.git"
+bug-reports: "https://github.com/abenori/satysfi-pagestyle/issues"
+depends: [
+  "satysfi" { >= "0.0.5" }
```
Comment (na4zagin3) [inline packages/satysfi-pagestyle-doc/satysfi-pagestyle-doc.1.0.0/opam:13]: Can you specify the upper bound?

```suggestion
  "satysfi" { >= "0.0.5" & < "0.0.7" }
```

File: packages/satysfi-pagestyle-doc/satysfi-pagestyle-doc.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "satyrographos" { >= "0.0.2.6" & < "0.0.3" }
+  "satysfi-dist"
+
+  "satysfi-pagestyle"
```
Comment (na4zagin3) [inline packages/satysfi-pagestyle-doc/satysfi-pagestyle-doc.1.0.0/opam:17]: Can you specify the version constraint of the main library?

```suggestion
  "satysfi-pagestyle" {= "%{version}%"}
```

File: packages/satysfi-pagestyle/satysfi-pagestyle.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "A SATySFi package for patgestyle"
+description: """
+Brilliant description comes here.
```
Comment (na4zagin3) [inline packages/satysfi-pagestyle/satysfi-pagestyle.1.0.0/opam:4]: Do you mind to add a short description here or remove the `description` field?

**Review by na4zagin3 — CHANGES_REQUESTED** [review]

File: packages/satysfi-pagestyle/satysfi-pagestyle.1.0.0/opam [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+bug-reports: "https://github.com/abenori/satysfi-pagestyle/issues"
+depends: [
+  "satysfi" { >= "0.0.5" }
+  "satyrographos" { >= "0.0.2.6" & < "0.0.3" }
+  "satysfi-dist"
+
+  # If your library depends on other libraries, please write down here
+  "satysfi-base" { >= "1.4.0" & < "2.0.0" }
+  "satysfi-fss" { >= "0.2.0" & < "0.3.0" }
+  "satysfi-pagenumber"
```
Comment (na4zagin3) [inline packages/satysfi-pagestyle/satysfi-pagestyle.1.0.0/opam:20]: Can you specify version constraints?

```suggestion
  "satysfi-pagenumber" { >= "1.0.0" & < "2.0.0" }
```

**Comment by abenori** [comment]
Thank you. I modified them,

**Review by na4zagin3 — APPROVED** [review]

**Comment by na4zagin3** [comment]
LGTM.  Let me reopen the PR to rerun the CI.

**Comment by na4zagin3** [comment]
Let me reopen the PR to rerun the CI.

**Comment by na4zagin3** [comment]
Thanks!

**Pull request merged**